packages/wallet/src/__stories__/dummy-wallet-states.ts

```diff
@@ -47,6 +47,8 @@ const defaults = {
   uid: 'uid',
   processStore: {},
```

Comment:
The surrounding object strikes me as something that could be exported from some test file, and referenced elsewhere.

Reply:
I'm not sure we'd want to export an empty wallet state that has faked values in it. We could probably switch stories to use some our tests states but I think that's beyond the scope of this PR.